cpu/samd21/include/periph_cpu.h

```diff
@@ -32,7 +32,7 @@ extern "C" {
  * @brief   Override the default initial PM blocker
  * @todo   Idle modes are enabled by default, deep sleep mode blocked
  */
-#define PM_BLOCKER_INITIAL  { .val_u32 = 0x00000001 }
+#define PM_BLOCKER_INITIAL      0x00000001
```

Comment:
This indent looks strange to me?

Reply:
I think it is now the same as further down in the file.

Reply:
Yes, nevermind. *gets more coffee